docs/developers/release.md

```diff
@@ -0,0 +1,106 @@
+
+# How to Create a new Release
+
+This document is intended for PHAC-NML developers for use when creating a new official release.
+
+Please follow all steps to ensure the release is the same across github / windows build / pypi
+
+### Git Release
+#### Changes
+
+Double check all the code in this release has been merged into development, and the CHANGELOG.md file reflects these changes with an appropriate Version Number update.
+
+Make sure your local development branch is up to date with origin
+
+#### Update Version Number
+
+Execute the following command to update version number in the various build files
+
+    ./scripts/update.sh <X.Y.Z>
+
+For Major updates, that completely break compatibility, update to a new X version
+
+For Feature Updates, update to a new Y version
+
+For Bug Fixes and Minor Updates, update to a new Z version
+
+#### Create a new tag
+
+Create a new tag with the same version number as above
+
+    git tag X.Y.Z
+
```

Comment:
Add a check here for the user to make sure the tag actually got created. I think it's something like `git tag`

Reply:
Good idea, added.